app/models/time_sheet.rb

```diff
@@ -19,27 +19,39 @@
 #  last_modified         :date
 #  status                :integer          default(0)
 #  signer                :integer
+#  wimi_signed           :boolean          default(FALSE)
+#  hand_in_date          :date
 #
 
 class TimeSheet < ActiveRecord::Base
-  belongs_to :user
-  belongs_to :project
+    belongs_to :user
+    belongs_to :project
+    enum status: [ :pending, :accepted, :rejected]
 
-  validates :workload_is_per_month, inclusion: {in: [true, false]}
-  validates :salary_is_per_month, inclusion: {in: [true, false]}
+    validates :workload_is_per_month, inclusion: { in: [true, false] }
+    validates :salary_is_per_month, inclusion: { in: [true, false] }
 
-  def self.time_sheet_for(year, month, project, user)
```

Comment:
Anscheinend doch (ich hab die Methode leider nicht geschrieben), weil ansonsten der Aufruf in der self.time_sheet_for Methode nicht funktioniert. Als Fehler wird dann undefined method ausgegeben